user-console/src/main/resources/org/pentaho/mantle/public/themes/ruby/mantleRuby.css

```diff
@@ -1054,18 +1054,69 @@ div#customDropdownPopupMajor .gwt-MenuBar-vertical .gwt-MenuItem:hover:not(.gwt-
 }
 
 #generated-content-location {
-  width: 350px;
+  /*width: 350px;*/
   border-radius: 4px 0 0 4px;
 }
 
 .schedule-label {
-  margin-top: 15px;
-  margin-bottom: 5px;
+  margin-top: unset;
+  margin-bottom: unset;
   color: #666666;
 }
 
+.new-schedule-dialog .schedule-dialog-content > table > tbody {
+  gap: var(--dialog-vertical-gap);
+}
+
+.schedule-output-location-dialog > div > * {
+  height: auto !important;
+}
+
+.schedule-output-location-dialog .dialog-content > table > tbody {
+  gap: var(--dialog-vertical-gap);
+}
+
+.schedule-output-location-dialog .dialog-content > table > tbody > tr {
+  flex: none;
+}
+
 #schedule-name-input {
-  width: 250px;
+  width: unset;
+  margin-bottom: unset;
+}
+
+.schedule-timestamp-listbox {
+  margin-left: unset;
+  margin-bottom: unset;
+  min-width: 200px;
+  width: unset;
+}
+
+.schedule-name-panel > tbody > tr {
+  flex-wrap: wrap;
+  gap: var(--dialog-vertical-gap);
```

Comment:
Variable no longer exists.
The equivalent would be:
```css
gap: var(--layout-gap-v) var(--layout-gap-h);
```

But please, check it this is still needed.

Reply:
Using the equivalent is having no effect in separating elements unlike previous change.